docs/demo/gaiden-css/components/picture/cover.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+  <style>
+    body {
+      background-color: #eff4fa;
+    }
+  </style>
+</head>
+
+<body>
+  <picture class="picture picture--cover">
+    <source srcset="/demo/gaiden-css/images/background-header-desktop.jpg" media="(min-width: 1200px)">
+    <source srcset="/demo/gaiden-css/images/background-header-large.jpg" media="(min-width: 960px)">
+    <source srcset="/demo/gaiden-css/images/background-header-medium.jpg" media="(min-width: 768px)">
+    <source srcset="/demo/gaiden-css/images/background-header.jpg" media="(max-width: 767px)">
+    <img src="/demo/gaiden-cssimages/background-header-medium.jpg" class="picture__image" alt="Imagem">
```

Comment:
@thiagodelmotte this file is just a code sample. The objective of alt here is to force the importance of this attribute.